frontend/src/components/core/Block/Block.tsx

```diff
@@ -124,32 +124,33 @@ const BlockNodeRenderer = (props: BlockPropsWithWidth): ReactElement => {
 const ChildRenderer = (props: BlockPropsWithWidth): ReactElement => {
   return (
     <>
-      {props.node.children.map(
-        (node: AppNode, index: number): ReactElement => {
-          // Base case: render a leaf node.
-          if (node instanceof ElementNode) {
-            // Put node in childProps instead of passing as a node={node} prop in React to
-            // guarantee it doesn't get overwritten by {...childProps}.
-            const childProps = { ...props, node: node as ElementNode }
-
-            const key = getElementWidgetID(node.element) || index
-            return <ElementNodeRenderer key={key} {...childProps} />
+      {props.node.children &&
```

Comment:
The `props.node.children &&` is the only thing added, the rest hasn't been changed.